hangups/ui/__main__.py

```diff
@@ -31,18 +33,21 @@
         ('inactive_tab', 'underline', 'light green'),
         ('msg_date', 'dark cyan', ''),
         ('msg_sender', 'dark blue', ''),
+        ('msg_self', 'light blue', ''),
```

Comment:
It works on my end. this is what it looks like: https://i.imgur.com/e1Hw73f.png. It might depend on the terminal colours you use though.:
I ended up going with this, looks pretty nice with base16 terminal colours.

col-palette-colors=88
col-active-tab-fg=black
col-active-tab-bg=h12
col-inactive-tab-fg=black, standout
col-inactive-tab-bg=h8
col-msg-date-fg=h5
col-msg-sender-fg=h6, bold
col-msg-self-fg=h12
col-status-line-fg=h8, standout
col-status-line-bg=black
col-tab-background-fg=default